### DISCUSSION TITLE
Solved - Periodic errors in Metrics analysis - starting 2021-03-10:17 GMT.

### DISCUSSION THREAD

**Discussion by CodeImproverTestUser** [CodeImprover/CodeImprover#16] [Status on service]
There are currently some analysis randomly failing.
We are working to fix the problem 
Regards Anders Product Manager

**Comment by adjuc** [comment]
The problems should be solved. 
We have made an update with some bug fixes and better logging. 
Please do not hesitate to contact us if you experience any irregularities.
Regards Anders Djursaa Product Manager
